src/db/db.ts

```diff
@@ -51,5 +51,6 @@ const config = ProcessEnv.dbUrl
     }
 
 export type BaseProtocol<T = any> = pgPromise.IBaseProtocol<T>
+export type IDatabase<T = any> = pgPromise.IDatabase<T>
```

Comment:
Should we remove the I prefix to match other?

Reply:
where this is needed ?

Reply:
The type is needed in some comments
https://github.com/openforis/arena/pull/1468/files/af505e5b5e56630890923c47308aff25640fe40d#diff-0872008e800b44c5d35e0d94f257206bd7f4f23a325df93b9abf36768e5fb3f6R84